index.js

```diff
@@ -432,22 +439,22 @@ function convertBulletPointsToText(bullets) {
     return text;
 }
 
-function queueSendEmail() {
-    //SES means simple email service. It is the sending email api offered by Amazon Web Service
-    //TODO change to Gmail API
-    return new AWS.SES({apiVersion: "2010-12-01"})
-        .sendEmail(emailClient.generateParams(userName, data))
-        .promise()
-        .then(data => {
-            console.log(data.MessageId);
-            resetToInitialState();
-            context.done(null, "Success");
-        }).catch(err => {
-            console.error(err, err.stack);
-            resetToInitialState();
-            context.done(null, "Failed");
-        });
-}
+
+//function queueSendEmail() {
```

Comment:
Does putting the send email code into its own function not work? Is that the reason for commenting it out?

Reply:
that's the previous retired function the coops wrote. I'm not sure what was broken there we should just delete it at this point since we already have a working one.